### ISSUE TITLE
[request] option to disable the fzf preview window

### ISSUE THREAD

**Issue by m040601** [roosta/tmux-fuzzback#17]
TLDR:

Disable the fzf preview window by default and add an option like for ex:

```
set -g @fuzzback-fzf-previews-enabled 1
```

Hi,

long time user and fan of this extremely usefull and clever plugin. By the way, I also use Arch ...

I dont like that fzf preview window being forced by default. Too much clutter for my taste.
I also tend to use tmux-fuzzback to "bulk collect" items.
Could be urls, or "ids" or whatever that was displayed in my long tmux scroll back buffer.
I dont use it in "popup" mode, I use it in "split" mode in this case.
Here is a silly example, where I'm collection youtube "ChannelID"s, https://asciinema.org/a/4eeL2XwAvuf2IGi37xKu5H5TU

This allows me to filter and "shrink" my tmux scrollback buffer to only a list of items I'm interested in.  I then use tmux copy-mode to copy those items, all at once to a tmux-buffer. Very nice

The thing is that by default, tmux-fuzzback, has the preview option for fzf enabled.
So that on the right side there is a big block of text I dont need.

And I dont want this, as I dont need any context and dont want to preview those lines.
I just want the items in a nice list.

So I went ahead and edited "fuzzback.sh", to comment out that part

```
finder_split_cmd() {
  local fuzzback_finder="$3"
...
...
    --no-sort \
    # 2023-03-28 commented out to disable the fzf preview
    # --preview-window=nowrap \
    # --preview="$CURRENT_DIR/preview.sh $CAPTURE_FILENAME {}" \
    --print-query \
....
....
}

```

It seems to work without problems. But now I have changed and customized your script.

I would rather prefer if this was offered as an option, so that I can just install fuzzback without having to hack it.

Something like a boolean option, to enable/disable fzf previews, for example

```
set -g @fuzzback-fzf-previews-enable 1
```

And by default they are disabled, as I find it distracting.

Thanks in advance

**Comment by roosta** [comment]
Hi there fellow arch user :D, so glad that you're enjoying the plugin. I can for sure setup an option for that. I'll look at it later today if I get the chance. I think I'll leave it on by default, but an option can turn it of permanently.

**Comment by roosta** [comment]
Alright, got a branch going with the option, https://github.com/roosta/tmux-fuzzback/tree/feat/preview-toggle, mind giving it a go before I merge?

Details are in readme, but essentially there's a new option `fuzzback-enable-preview` that you need to set to 0 to disable it.

**Comment by roosta** [comment]
Added some more commits, renamed the option to `fuzzback-hide-preview` and you need to set it to `1` instead, made more sense that way.

**Comment by roosta** [comment]
Any interest in this at all? otherwise I'm closing this issue due to inactivity...

**Comment by amjerm** [comment]
@roosta I just tried the option in the branch mentioned above and it works for me. I'd definitely like this option to be merged as well.

Thanks for a great plugin!

By the way, when cloning the repo to try out that branch I noticed that the README has the clone command in the manual instructions listed as

```
git clone https://github.com/roosta/tmux-fuzzback
```

rather than

```
git clone https://github.com/roosta/tmux-fuzzback.git
```

**Comment by roosta** [comment]
Hi @amjerm, thanks for the readme heads up. I'll merge later today, thanks for checking it out!

**Comment by roosta** [comment]
Merged!

**Comment by amjerm** [comment]
Thank you!
